pkg/virt-launcher/virtwrap/converter/archconverter.go

```diff
@@ -40,6 +40,7 @@ type ArchConverter interface {
 	isUSBNeeded(vmi *v1.VirtualMachineInstance) bool
 	supportCPUHotplug() bool
 	isSMBiosNeeded() bool
+	isVMPortNeeded() bool
```

Comment:
I would maybe rename this to `hasVMPort`, because we ultimately want to turn it off if yes.

Reply:
ok